commercetools-convenience/src/main/java/io/sphere/sdk/sequencegenerators/BigIntegerNumberGenerator.java

```diff
@@ -0,0 +1,13 @@
+package io.sphere.sdk.sequencegenerators;
+
+import java.math.BigInteger;
+import java.util.concurrent.CompletionStage;
+
+/**
+ Defines how to implement an incremental number generator
```

Comment:
not very important, but write javadoc with the same format as other classes. Ex: https://github.com/commercetools/commercetools-jvm-sdk/blob/master/commercetools-models/src/main/java/io/sphere/sdk/carts/AnonymousCartSignInMode.java#L8

And explain why the function exists.
Example:
```java
/**
* Incremental number generator.
* This generator ensures that all calls (even in parallel) generates a different number.
 */
public interface BigIntegerNumberGenerator {
    CompletionStage<BigInteger> getNextNumber();
}
```